<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Behavior</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Editing</string>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="siBinPrefixDefaultBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When enabled, for properties that are typically expressed using binary SI prefixes (e.g. '1KiB' for 1024 Bytes), whenever the user enters a value with a non-binary SI prefix, it is instead interpreted as if the user entered the binary prefix (e.g. 1KB = 1024 Bytes and not 1000 Bytes).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Interpret SI prefixes as binary for binary properties</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Databases</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Startup Connection</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="connStartupBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="connList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="connAddButton">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-add">
                  <normaloff/>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="connRemoveButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-remove">
                  <normaloff/>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QFormLayout" name="formLayout">
              <property name="horizontalSpacing">
               <number>20</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Connection Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="connNameField">
                <property name="enabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="connTypeBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Connection Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QStackedWidget" name="connStackWidget">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="page_3">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QWidget" name="widget_6" native="true">
                    <layout class="QFormLayout" name="formLayout_3">
                     <property name="fieldGrowthPolicy">
                      <enum>QFormLayout::ExpandingFieldsGrow</enum>
                     </property>
                     <property name="horizontalSpacing">
                      <number>20</number>
                     </property>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="connServerField"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Port</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QSpinBox" name="connPortField">
                       <property name="maximum">
                        <number>999999</number>
                       </property>
                       <property name="value">
                        <number>3306</number>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>User Name</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLineEdit" name="connUserField"/>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>Host</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Database</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="connDbField"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_4">
                 <layout class="QFormLayout" name="formLayout_2">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::ExpandingFieldsGrow</enum>
                  </property>
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Database File</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QWidget" name="widget_7" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="connSqliteFileField"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="connSqliteChooseButton">
                       <property name="text">
                        <string>Choose...</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Embedded File Settings</string>
             </property>
             <layout class="QFormLayout" name="formLayout_5">
              <property name="horizontalSpacing">
               <number>20</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Root Path</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="widget_9" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="connFileRootField"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="connFileRootChooseButton">
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
